### ISSUE TITLE
Crash on optional chaining

### ISSUE THREAD

**Issue by mysticatea** [estools/esquery#110]
Some queries crash on optional chaining.

**Repro code:**

- Node 12.12.0
- `acorn@7.3.1`
- `esquery@1.3.1`

```js
const acorn = require("acorn")
const esquery = require("esquery")

const ast = acorn.parse("a?.b", { ecmaVersion: 2020 })
console.log(esquery(ast, ":nth-child(1)"))
```

**Error:**

```
TypeError: Cannot read property 'length' of undefined
    at f (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:32431)
    at l (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:30995)
    at f.enter (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:32935)
    at f.__execute (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:8604)
    at f.traverse (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:9487)
    at Object.y [as traverse] (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:2490)
    at h (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:32878)
    at d (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:33189)
    at m (C:\Users\t-nagashima.AD\dev\sandbox\node_modules\esquery\dist\esquery.min.js:1:33278)
```

**Comment by brettz9** [comment]
I've asked @michaelficarra at https://github.com/estools/esquery/issues/67#issuecomment-631170466 whether we might provide a method to supply a custom traverser or VisitorKeys (to support such more recent syntaxes) but pending such a decision, and as also mentioned there, #105 added a lite build which should at least allow overriding the traverser version in use.

FWIW, https://github.com/estools/estraverse/issues/112 was filed on the `estraverse` side, so if support gets added there, this particular feature at least may become available anyways (at least with a version bump if needed).

**Comment by brettz9** [comment]
I believe this can now be closed.

**Comment by michaelficarra** [comment]
Fixed by #112.
